include/ck_tile/ops/gemm.hpp

```diff
@@ -26,6 +26,7 @@
 #include "ck_tile/ops/gemm/kernel/batched_gemm_kernel.hpp"
 #include "ck_tile/ops/gemm/kernel/gemm_kernel.hpp"
 #include "ck_tile/ops/gemm/kernel/gemm_tile_partitioner.hpp"
+#include "ck_tile/ops/gemm/kernel/gemm_offset_block.hpp"
```

Comment:
run `python3 remod.py` under `include/ck_tile`, you may find we will sorting the headers before include into this gemm.hpp :) in case ppl argue about alphbetical order